packages/standards/src/PostInstall.php

```diff
@@ -0,0 +1,94 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * Executes after installing via Composer.
+ *
+ * @author kraftbj
+ * @package Automattic/jetpack-standards
+ */
+
+namespace Automattic\Jetpack\Standards;
+
+use Composer\Composer;
+use Composer\IO\IOInterface;
+use Composer\Plugin\PluginInterface;
+use Composer\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Post Install class.
+ *
+ * @package Automattic\jetpack-standards
+ */
+class PostInstall implements PluginInterface, EventSubscriberInterface {
+	/**
+	 * Composer Plugin activation.
+	 *
+	 * Unused. Intended for changing Composer internals, which we don't need.
+	 *
+	 * @param Composer    $composer Composer.
+	 * @param IOInterface $io       IOInterface.
+	 */
+	public function activate( Composer $composer, IOInterface $io ) {
+	}
+
+	/**
+	 * Copies standards and GitHub templates from this plugin to the root directory of the included projects.
+	 *
+	 * Currently assumes that this file is at the third level of root.
+	 */
+	public static function post_install() {
+		self::xcopy( ( __DIR__ ) . '/standards', dirname( dirname( dirname( dirname( __DIR__ ) ) ) ) );
+		self::xcopy( ( __DIR__ ) . '/github', dirname( dirname( dirname( dirname( __DIR__ ) ) ) ) . '/.github' );
```

Comment:
I wonder if the root package directory can be grabbed from the `Composer\Script\CommandEvent` instance that's provided as an argument to `post_install`.